edit_book.php

```diff
@@ -0,0 +1,44 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The current implementation directly uses raw SQL queries with user input, which is vulnerable to SQL injection attacks.

Recommendations:
- **Use Prepared Statements**: Utilize prepared statements to prevent SQL injection. This can be done using either the `mysqli` or `PDO` extension.

Example:
```php
$query = "UPDATE books SET title = ?, author = ?, year = ? WHERE id = ?";
$stmt = $connection->prepare($query);
$stmt->bind_param("ssii", $title, $author, $year, $id);
$stmt->execute();
```